bokehjs/src/coffee/widget/slider.coffee

```diff
@@ -16,6 +17,7 @@ class SliderView extends ContinuumView
     html = @template(@model.attributes)
     @$el.html(html)
     @render()
+    dom_util.add_css_classes(@$el, @model.css_classes())
```

Comment:
However, as `css_classes` is a property of `Widget`, this logic should have been implemented in `WidgetView`. There is no such class, so you would have to implement one, make widget views inherit from it, rename rendering functions, etc.